lanelet2_core/include/lanelet2_core/Exceptions.h

```diff
@@ -2,6 +2,10 @@
 #include <stdexcept>
 #include <vector>
 
+#ifdef __QNXNTO__
```

Comment:
Just unconditionally include `<string>`. It seems like this include is missing anyways.